### ISSUE TITLE
Change in POST URL 

### ISSUE THREAD

**Issue by kcaponeAP** [activeprospect/leadconduit-integration-datamyx#6]
We received an email from Thanh @ Datamyx. 

https://activeprospect.zendesk.com/agent/tickets/7793

It appears that they will be changing the structure of the URL we use to POST to their API. I have asked them a little bit more about the change. Specifically what 'somesite' will be replaced with. Will it be company/account name? Will update shortly. Wanted to get some attention on this since the change may be by end of July 2015.

"You are currently calling https://intelidataexpress.datamyx.com/webservice/application.asmx (https://intelidataexpress.datamyx.com/webservice/application.asmx)

We are going to be changing the location of our webservice to be something like https://somesite.datamyx.com/webservice/application.asmx. Do you have any idea of the level of effort from your side to make this change? We’re planning on making the location change by the end of the month."

**Comment by alexkwolfe** [comment]
How are they planning the cut-over? Will there be some period of time during which both are supported?

**Comment by kcaponeAP** [comment]
Checking on that for you @alexkwolfe - they just stated:

" It will most likely be HTTPS://legacy.datamyx.com/"

I need more concrete info from them.

**Comment by kcaponeAP** [comment]
"There will be a period of time next week where both will run in parallel.

Not sure how the cut over will affect users if at all. If you are able to repoint your call’s location, the username/password and call itself should remain exactly the same."

**Comment by alexkwolfe** [comment]
There should be a period of time when we can post to both https://intelidataexpress.datamyx.com/webservice/application.asmx and https://somesite.datamyx.com/webservice/application.asmx. 

If they expect us to use https://legacy.datamyx.com/webservice/application.asmx, then there will need to be a period of time when we can post to https://intelidataexpress.datamyx.com/webservice/application.asmx and https://legacy.datamyx.com/webservice/application.asmx.

**Comment by kcaponeAP** [comment]
Update from Datamyx:

"We’re in the process of getting https://legacy.datamyx.com right now. Hopefully I will be able to give you the URL by today or tomorrow."

I'll be back in touch when they post their reply to me!

**Comment by alexkwolfe** [comment]
I don't understand what they are saying.

**Comment by kcaponeAP** [comment]
Well that's datamyx stating that they are in the process of getting the new URL to POST to up and running. The second part is unclear, but this is heads up that we should be moving our POST URL for the Datamyx integration to the new URL "https://legacy.datamyx.com" this week.

**Comment by kcaponeAP** [comment]
Datamyx has finished the changeover process for their new system. Can we please prioritize making the change @alexj 

Here is the most recent email from them:

"Hey Karalyn,

We’ve finally gotten our new site up. The address will be https://legacy.datamyx.com The URLs that you will need to change are the following:

Change the Current URL in use to:

New URL

https://intelidataexpress.datamyx.com/webservice/application.asmx
https://legacy.datamyx.com/webservice/application.asmx
https://intelidataexpress.datamyx.com/webservice/lookup.asmx
https://legacy.datamyx.com/webservice/lookup.asmx

The user name and password should be exactly the same as when you were connecting to our old site.

Just to confirm these are the IP addresses that we have whitelisted for you. 

https://support.activeprospect.com/hc/en-us/articles/201326209-What-are-the-IP-addresses-of-the-servers-that-deliver-leads- (https://support.activeprospect.com/hc/en-us/articles/201326209-What-are-the-IP-addresses-of-the-servers-that-deliver-leads-)

Do you have an estimate on the amount of work it would take on your side to make this happen?

Thank you again. Please give me a call or email if you have any questions.

Thanh Khong

DATAMYX™

Application Development Manager"

###

If you have any questions, please connect with their tech directly at the email address below:
https://activeprospect.zendesk.com/agent/tickets/7793

**Comment by alexj** [comment]
Yes. @ros3bud can you work on this while you're waiting to hear back from Google?

**Comment by alexkwolfe** [comment]
Why did he give 4 new URLs? I don't get what we're supposed to do. Are we just changing the hostname from intellidataexpress.datamyx.com to legacy.datamyx.com?

**Comment by alexj** [comment]
My impression is that there was a formatting issue when he sent the email. So
https://intelidataexpress.datamyx.com/webservice/application.asmx should change to https://legacy.datamyx.com/webservice/application.asmx 
and 
https://intelidataexpress.datamyx.com/webservice/lookup.asmx changes to 
https://legacy.datamyx.com/webservice/lookup.asmx

**Comment by alexkwolfe** [comment]
That's what it seems like to me too.

**Comment by ros3bud** [comment]
Closing this issue out as contact at Datamyx said it is no longer needed.

**Comment by alexkwolfe** [comment]
No longer needed at all? Like they're not switching URLs any more?

**Comment by alexj** [comment]
I expect that they will still change the URL, once they figure out the issue with whitelisting our IPs.

**Comment by ros3bud** [comment]
From Thanh at Datamyx: 

Changing gears a little bit.  Since we’re having so much trouble getting the connection working we’re going to leave the old webservice up for the time being that’s running on production.  We won’t need any changes from you for right now.  I’ll reach out again if that changes.
